templates/hpe/index.html

```diff
@@ -58,10 +60,10 @@ <h2>A single point of contact for technical expertise</h2>
 <section class="p-section">
   <hr class="is-fixed-width p-rule">
   <div class="row">
-    <div class="col-6 col-medium-3">
+    <div class="col-6 col-medium-6 p-block">
```

Comment:
Now we have blocks on cols?

Reply:
this is wrong (although it doesn't cause problems). Ideally should be on a child element.